shopinvader_sale_coupon/services/abstract_sale.py

```diff
@@ -15,20 +15,11 @@ def _convert_one_sale(self, sale):
         res = super()._convert_one_sale(sale)
         res.update(
             {
-                "promo_code": sale.promo_code,
```

Comment:
Since several promo codes can be applied, we can now return a string that is a list of the applied promo codes (and an empty string if no promo code is applied). It will minimize the difference in the services.